### PR TITLE
feat(plugins): Wire up PluginConfiguration and PluginSdks for v2 framework

### PR DESCRIPTION
**Pull request by robzienert**

### CODE REVIEW

File: kork-plugins/src/main/kotlin/com/netflix/spinnaker/kork/plugins/v2/context/PluginSdksRegisteringCustomizer.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import com.netflix.spinnaker.kork.plugins.sdk.SdkFactory
+import org.pf4j.Plugin
+import org.springframework.context.ConfigurableApplicationContext
+import org.springframework.context.support.GenericApplicationContext
+
+/**
+ * Registers [PluginSdks] into a [Plugin] application context.
+ *
+ * V1 API compatibility. We could get rid of the PluginSdks concept entirely now and just inject interfaces for
+ * the SDKs we want to expose, rather than proxying them through [PluginSdksImpl].
```
Comment (claymccoy) [inline kork-plugins/src/main/kotlin/com/netflix/spinnaker/kork/plugins/v2/context/PluginSdksRegisteringCustomizer.kt:29]: Nice!

**Comment by robzienert** [comment]
Tests are in.

File: kork-plugins-tck/src/main/kotlin/com/netflix/spinnaker/kork/plugins/PluginFrameworkTest.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.kork.plugins
+
+import com.netflix.spinnaker.config.PluginsAutoConfiguration
+import dev.minutest.junit.JUnit5Minutests
+import org.springframework.context.annotation.Import
+
+abstract class PluginFrameworkTest<T : PluginFrameworkTestFixture> : JUnit5Minutests {
```
Comment (jonsie) [inline kork-plugins-tck/src/main/kotlin/com/netflix/spinnaker/kork/plugins/PluginFrameworkTest.kt:22]: Looks like this isn't being used anywhere in this PR.

Comment (robzienert) [reply kork-plugins-tck/src/main/kotlin/com/netflix/spinnaker/kork/plugins/PluginFrameworkTest.kt:22]: Right you are! Oops.

**Pull request merged**